ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_service_insertion.schema.yml

```diff
@@ -8,7 +8,41 @@ type: dict
 keys:
   router_service_insertion:
     type: dict
-    description: Configure network services inserted to data forwarding
+    description: Configure network services inserted to data forwarding.
     keys:
       enabled:
         type: bool
+      connections:
+        type: list
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              description: Configure name of the network service.
+            interface:
+              type: dict
+              description: Interface to be used to reach the network service.
```

Comment:
So it seems that ethernet and tunnel are mutually exclusive:

```
edge2(config-connection-BLAH)#show active
router service-insertion
   connection BLAH
      interface Tunnel42 primary
      interface Tunnel43 secondary
edge2(config-connection-BLAH)#interface ?
  Ethernet  Ethernet Sub Interface
  Tunnel    Tunnel interface
edge2(config-connection-BLAH)#interface Ethernet 3 next-hop 1.1.1.1
edge2(config-connection-BLAH)#show active
router service-insertion
   connection BLAH
      interface Ethernet3 next-hop 1.1.1.1
edge2(config-connection-BLAH)#interface tunnel 42 secondary
edge2(config-connection-BLAH)#show active
router service-insertion
   connection BLAH
      interface Tunnel42 secondary
```

at the minomum we should mention it in description and say which one we will choose if both are configured

Reply:
Mentioned in description that ethernet is chosen if we mention both ethernet and tunnel.